packages/mds-db/policies.ts

```diff
@@ -56,6 +56,22 @@ export async function readPolicies(params?: {
   return res.rows.map(row => row.policy_json)
 }
 
+export async function readActivePolicies(timestamp: Timestamp = now()): Promise<Policy[]> {
+  const client = await getReadOnlyClient()
+  const conditions = []
+  const vals = new SqlVals()
+  conditions.push(`policy_json->>'start_date' <= ${vals.add(timestamp)}`)
+  //  conditions.push(`policy_json->>'end_date' >= ${vals.add(timestamp)}`)
+  const sql = `select * from ${schema.TABLE.policies} WHERE ${conditions.join(' AND ')}`
+  const values = vals.values()
+  const res = await client.query(sql, values)
+  return res.rows
+    .map(row => row.policy_json)
+    .filter(policy => {
+      return policy.end_date === null || policy.end_date >= timestamp
+    })
```

Comment:
Seems super strange to me that we do the `start_date` check as part of the query, but then do a `filter` on the result for `end_date`

Reply:
Noted. I'm keeping `readActivePolicies` though, since `readPolicies` is a more generalized method that allows you to set `get_unpublished` to `true`, and it feels weird to have that available as an option when active policies must be published.